supporting_scripts/course-scripts/quick-course-setup/create_course.py

```diff
@@ -90,14 +87,17 @@ def create_course(session):
     if response.status_code == 201:
         logging.info(f"Created course {course_name} with shortName {course_short_name} \n {response.json()}")
     else:
-        logging.error("Problem with the group 'students' and interacting with a test server? Is 'is_local_course' in "
-                      "'config.ini' set to 'False'?")
+        logging.error("Problem with the group 'students' and interacting with a test server? "
+                      "Is 'is_local_course' in 'config.ini' set to 'False'?")
         raise Exception(
-            f"Could not create course {course_name}; Status code: {response.status_code}\n Double check whether the courseShortName {course_short_name} is not already used for another course!\nResponse content: {response.text}")
-    return response
+            f"Could not create course {course_name}; Status code: {response.status_code}\n"
+            f"Double check whether the courseShortName {course_short_name} is not already used for another course!\n"
```

Comment:
I think we could improve the error message here, I ran into the problem while testing 😅 
```suggestion
            f"Double check whether the courseShortName {course_short_name} is valid (e.g. no special characters such as '-') not already used for another course!\n"
```